src/map.cpp

```diff
@@ -533,7 +533,7 @@ void ServerMap::transformLiquids(std::map<v3s16, MapBlock*> &modified_blocks,
 		infostream<<"transformLiquids(): initial_size="<<initial_size<<std::endl;*/
 
 	// list of nodes that due to viscosity have not reached their max level height
-	std::deque<v3s16> must_reflow;
+	std::vector<v3s16> must_reflow;
```

Comment:
Why? Deque is more efficient for insertions.

Reply:
@SmallJoker,

- `must_reflow` as `std::vector`

https://stackoverflow.com/questions/35483716/is-stddeque-faster-than-stdvector-for-inserting-at-the-end#answer-42743181